src/Symfony/Contracts/Translation/TranslatorTrait.php

```diff
@@ -46,90 +44,6 @@ public function trans($id, array $parameters = array(), $domain = null, $locale
         return strtr((string) $id, $parameters);
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function transChoice($id, $number, array $parameters = array(), $domain = null, $locale = null)
```

Comment:
This function was removed. It was a replacement for the deprecated `MessageSelector`. I do not think it is needed anymore, right?